src/api/composition/CompositionCollection.js

```diff
@@ -272,6 +272,19 @@ define([
         this.remove(child, true);
     };
 
+    CompositionCollection.prototype._synchronize = function () {
+        this.mutationListener = this.publicAPI.objects.observe(this.domainObject, '*', (newDomainObject) => {
+            this.domainObject = JSON.parse(JSON.stringify(newDomainObject));
+        });
+    };
+
+    CompositionCollection.prototype._destroy = function () {
+        if (this.mutationListener) {
+            this.mutationListener();
+        }
+        delete this.mutationListener;
+    };
```

Comment:
Technically it doesn't matter - it won't throw an error if `this.mutationListener` doesn't exist, but it might be semantically clearer to put it in the it statement. I'll move it.